pkg/steps/multi_stage/gen.go

```diff
@@ -57,7 +57,7 @@ type generatePodOptions struct {
 func defaultGeneratePodOptions() *generatePodOptions {
 	return &generatePodOptions{
 		IsObserver:       false,
-		NodeArchitecture: string(api.NodeArchitectureAMD64),
+		NodeArchitecture: "",
```

Comment:
But we want to default it to amd64. Is there any particular reason of removing it?

Reply:
I missed adding back after testing, nice catch